XamlControlsGallery/ControlPages/CornerRadiusPage.xaml

```diff
@@ -0,0 +1,75 @@
+            <Run FontWeight="bold" Text="2px" />
+            <Run Text="." />
+            <LineBreak />
+            <LineBreak />
+            <Run Text="For transient UI elements that appear on screen temporarily, like MenuFlyout, or elements that overlay other UI, like TabView tabs," />
+            <LineBreak />
+            <Run Text="the default radius value we use for is" />
+            <Run FontWeight="Bold" Text="4px" />
+            <Run Text="." />
+        </TextBlock>
+        <TextBlock>
+            <Run Text="These demos require Windows 10 Version 1809 or later to work properly. To set a conditional namespace, check out our" />
+            <Hyperlink
+                NavigateUri="https://docs.microsoft.com/en-us/windows/uwp/debug-test-perf/conditional-xaml#:~:text=%20Conditional%20XAML%20%201%20Conditional%20namespaces.%20To,conditional%20XAML%2C%20but%20you%20can%20also...%20More%20"
+                TextDecorations="None"
+                ToolTipService.ToolTip="https://docs.microsoft.com/en-us/windows/uwp/debug-test-perf/conditional-xaml#:~:text=%20Conditional%20XAML%20%201%20Conditional%20namespaces.%20To,conditional%20XAML%2C%20but%20you%20can%20also...%20More%20">
+                <Run Text="docs" />
+            </Hyperlink>
+        </TextBlock>
+        <local:ControlExample
+            x:Name="Example1"
+            HeaderText="A button with CornerRadius"
+            MinimumUniversalAPIContract="1809">
+            <Button Windows10version1809:CornerRadius="5" Content="MyButton" />
+            <local:ControlExample.Xaml>
+                <x:String>
+                    &lt;Button CornerRadius=&quot;5&quot; Content=&quot;MyButton&quot;/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+        </local:ControlExample>
+        <local:ControlExample
+            x:Name="Example2"
+            HeaderText="A button without CornerRadius"
+            MinimumUniversalAPIContract="1809">
+            <Button Windows10version1809:CornerRadius="0" Content="MyButton" />
+            <local:ControlExample.Xaml>
+                <x:String>
+                    &lt;Button CornerRadius=&quot;0&quot; Content=&quot;MyButton&quot;/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+        </local:ControlExample>
+        <local:ControlExample x:Name="Exaple3" MinimumUniversalAPIContract="1809">
+            <Button Windows10version1809:CornerRadius="5,0,5,0" Content="CornerRadius: 5,0,5,0" />
+        </local:ControlExample>
+        <local:ControlExample x:Name="Exaple4" MinimumUniversalAPIContract="1809">
+            <Button Windows10version1809:CornerRadius="5,5,0,0" Content="CornerRadius: 5,5,0,0" />
+        </local:ControlExample>
+
+        <TextBlock>
+            <Run Text="For more, visit " />
+            <Hyperlink
+                NavigateUri="https://docs.microsoft.com/en-us/windows/uwp/design/style/rounded-corner"
```

Comment:
Please remove 'en-us' from the URLs; this will help direct people to the relevant page for their language.